test-suite/Makefile

```diff
@@ -33,10 +33,10 @@ include ../Makefile.common
 #######################################################################
 
 ifneq ($(wildcard ../_build),)
-BIN:=$(shell cd ..; pwd)/_build/install/default/bin/
-COQLIB:=$(shell cd ..; pwd)/_build/install/default/lib/coq
+BIN:='$(shell cd ..; pwd)'/_build/install/default/bin/
```

Comment:
I suggest defining a new variable, say ROOT, that includes the `'` and there write a comment explaining that it is a make thing and eval is necessary later to make it shell compatible.

Reply:
@snyke7: I'll assume that you're ok with the suggestion of @gares and I'll push a commit implementing it, then merge tomorrow.

Thanks again for the fix!